docs/sanity-migration/11-neon-integration-mrm.md

```diff
@@ -0,0 +1,304 @@
+# Chapter 11: Neon Integration (Modern Rock Madness)
+
+## Overview
+
+This chapter documents the hybrid Sanity + Neon architecture for **Modern Rock Madness** (MRM). This architecture splits tournament data between two systems:
+
+- **Sanity CMS**: Staff-managed configuration (tournament setup, bands, brackets, matches)
+- **Neon PostgreSQL**: High-volume user data (votes for each matchup)
+
+This separation provides:
+- **Cost efficiency**: Thousands of votes don't impact Sanity API costs
+- **Performance**: PostgreSQL handles vote aggregation efficiently
+- **Real-time updates**: Fast vote counting for live bracket updates
+- **Scalability**: Database can handle high traffic during voting periods
+
+### Data Flow
+
+```
+┌─────────────┐         ┌──────────────┐         ┌─────────────┐
+│   Sanity    │◄────────│  Next.js API │────────►│    Neon     │
+│             │         │              │         │  PostgreSQL │
+│ • Tournament│         │ • Validate   │         │             │
+│   Config    │         │ • Authorize  │         │ • Votes     │
+│ • Bands     │         │ • Process    │         │ • Users     │
+│ • Matches   │         │              │         │             │
+└─────────────┘         └──────────────┘         └─────────────┘
+```
+
+1. User visits bracket → Frontend fetches tournament/match data from Sanity
+2. User votes in matchup → API validates match is active
+3. API stores vote in Neon PostgreSQL
+4. Vote counts aggregated from Neon in real-time
+5. Winner determined and updated in Sanity after match closes
+
+---
+
+## Sanity Schemas
+
+### `mrmTournament`
```

Comment:
As with yep, I don't want to see "mrm" as a prefix or any other part of a name. Spell out modern_rock_madness or modern-rock-madness, whatever is most appropriate